Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md

```diff
@@ -0,0 +1,55 @@
+---
+uid: project-wide-actions
+---
+# Project-Wide Actions
+
+The Input System stores your configuration of [Input Actions](Actions.md) and their associated [Bindings](ActionBindings.md), [Action Maps](ActionsEditor.html#configure-action-maps) and [Control Schemes](ActionBindings.md#control-schemes) in an [Action Asset](ActionAssets.md) file.
+
+While it's possible to have more than one Action Asset in a project, most projects only ever need a single Action Asset. This is because an Action Asset can contain multiple [Action Maps](ActionsEditor.html#configure-action-maps), which each containing a set of actions relevant to the various parts of your project (such as UI navigation, Gameplay, etc).
+
```

Comment:
Could be more specific in this paragraph and mention that it is available as a "preloaded asset" which is loaded at player start-up and kept alive until the player terminates. See https://docs.unity3d.com/ScriptReference/PlayerSettings.GetPreloadedAssets.html

Reply:
fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)